content/introduction/concepts.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Concepts"
+description: "Learn more about the core concepts and domain model that make up the SweetOps methodology."
+weight: 3
+---
+
+SweetOps is built on top of a number of high-level concepts and terminology that are critical to understand prior to getting started. In this document, we break down these concepts to help you get a leg up prior to going through your first tutorial.
```

Comment:
```suggestion
SweetOps is built on top of a number of high-level concepts and terminology that are critical to understanding prior to getting started. In this document, we break down these concepts to help you get a leg up prior to going through your first tutorial.
```

Reply:
@osterman this reads wrong to me so I didn't include it. Did you mean "critical to understanding *SweetOps* prior to getting started"?  Either way, I feel like the original is better, but I'm open to a different sentence all together if it doesn't feel like it's working for you.

Reply:
All I did was correct `understand` → `understanding` (based on [Grammarly](https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen?hl=en))

Reply:
Also, there's a plugin for vscode.